ChangeLog

```diff
@@ -247,6 +247,8 @@ Changes
    * Clarify the documentation of mbedtls_ssl_setup.
    * Use (void) when defining functions with no parameters. Contributed by
      Joris Aerts. #678
+   * Update Win32 APIs used to remove those not permitted by the Windows Store.
+     Fix provided by Kevin Kane, Microsoft. #635
```

Comment:
Git has expertly added this as a "Changes" entry to "mbed TLS 2.8.0". This needs fixing.

Reply:
I'll fix that in gatekeeping. It's not worth re-running the CI to fix it in the PR.

Reply:
@Patater Git doesn't know how to merge changelog entries (not without [help](https://github.com/ARMmbed/mbedtls/pull/1195) anyway) so it's routine for us to correct the location of added entries when merging.

Reply:
Note to gatekeeper: Please fix this.

This can easily go unnoticed in gatekeeping as it doesn't cause a conflict. I'd recommend making the change in the PR, but if that can't be done for some reason, then please keep this fix in mind when gatekeeping.